CHT/HeatFlux.C

```diff
@@ -8,18 +9,20 @@ using namespace Foam;
 
 preciceAdapter::CHT::HeatFlux::HeatFlux
 (
-    const Foam::fvMesh& mesh,
-    const std::string nameT
-)
-:
-T_(
-    const_cast<volScalarField*>
-    (
-        &mesh.lookupObject<volScalarField>(nameT)
-    )
-)
+        const Foam::fvMesh& mesh,
+        const std::string nameT
+        )
+    :
+      T_(
+          const_cast<volScalarField*>
+          (
+              &mesh.lookupObject<volScalarField>(nameT)
+              )
+          ),
+      mesh_(mesh)
 {
     dataType_ = scalar;
+
 }
 
 void preciceAdapter::CHT::HeatFlux::write(double * buffer)
```

Comment:
@MakisH Here, we need to add the information of our boolean option `provideMeshConnectivity`, in order to steer the interpolation. Do you have any suggestions how to handle it. We need it also in the Temperature write function. Additionally, the current setup assumes a consistent mapping (the connectivity is provided for the write data). In a more general case, we should maybe add another option, which associates the data set and the direction.